spec/configParser.spec.js

```diff
@@ -202,4 +202,17 @@ describe("Should parse confkey from command-line", function () {
     ]);
   });
 
+  it('Should refer to another profile', function () {
```

Comment:
can you please  add here (or move) the test that covers the simpler case "profile: integration" ?